src/DependencyInjection/PimExtendedMeasureExtension.php

```diff
@@ -21,9 +21,27 @@ public function load(array $config, ContainerBuilder $container)
     {
         $measuresConfig = [];
 
-        $configDirectory = __DIR__ . '/../Resources/config/measures';
+        foreach ($container->getParameter('kernel.bundles') as $bundle) {
+            $reflection = new \ReflectionClass($bundle);
+            if (is_dir($configDirectory = dirname($reflection->getFileName()).'/Resources/config/measures')) {
+                $bundleConfig = $this->parseBundleMeasures($configDirectory);
+                $measuresConfig = array_replace_recursive($measuresConfig, $bundleConfig);
+            }
+        }
+
+        $coreConfig = $container->getParameter('akeneo_measure.measures_config');
+        $measuresConfig = array_replace_recursive($coreConfig, $measuresConfig);
+        $container->setParameter('akeneo_measure.measures_config', $measuresConfig);
+
+        $loader = new YamlFileLoader($container, new FileLocator(__DIR__.'/../Resources/config'));
+        $loader->load('services.yml');
+    }
+
+    protected function parseBundleMeasures($configDirectory)
```

Comment:
It should be in parseBundleMeasures method to avoid to duplicate too many things to get back this possibility. I would reuse it for CNET at least.
But maybe it could be done later ;)